src/general/OptionsMenu.tscn

```diff
@@ -193,6 +193,7 @@ custom_constants/margin_left = 15
 custom_constants/margin_bottom = 15
 
 [node name="Graphics" type="Control" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer"]
+visible = false
```

Comment:
Is this a problematic change? I think it is.
@Kasterisk can probably confirm.

Reply:
I don't know if this is what you are talking about but I think this problem is when I launch godot to  see the button I make invisible graphics and visible misc and when I leave Godot I forgot to make i back  so now it's done

Reply:
Yes, the initial tab shown should be Graphics tab.

Reply:
I don't know If it's about this you are talking about but I think the problem come's when I leave godot I make invisible graphics and visible misc and now I revert It into a visible graphics and an invisible misc

Reply:
Setting the visibilities back in the godot editor to what they were should resolve this.